website/content/about/working-groups.md

```diff
@@ -11,3 +11,4 @@ Following working groups operate under the TAG Environmental Sustainability:
 | Name                | Scope and Goals            | Meeting Time                          |
 |---------------------|----------------------------|---------------------------------------|
 | [Green Reviews](https://github.com/cncf/tag-env-sustainability/tree/main/working-groups/green-reviews) | [Green Reviews Working Group Charter](https://github.com/cncf/tag-env-sustainability/tree/main/working-groups/green-reviews/charter.md) | [Green Reviews Working Group - Meetings and Contact](https://github.com/cncf/tag-env-sustainability/tree/main/working-groups/green-reviews/README.md#meetings-and-contact) |
+| Communications| [Communications Working Group Charter](https://docs.google.com/document/d/1dVHzNtJB07Nv2JN6ZUkmRflgq3RL9BbaURbgPnGENdk/edit) | [Communications Working Group Calendar Invite](https://tockify.com/cncf.public.events/detail/659/1697112000000) |
```

Comment:
Should these links be to the GitHub repo instead? Just like we did for the Green Reviews WG so that all of the information is aggregated in one place?

Reply:
Yes, but it can't be updated until the charter PR is accepted to update. So we can hold on until then.